include/config.h

```diff
@@ -107,10 +107,18 @@ struct ConfigStruct {
 
   // Vibration Output Mode
   VibrationMode vibration_mode;
+  
+  // clench stuff
+  int clench_pressure_sensitivity;
+  int clench_duration_threshold;
+
+  int autoEdgingDurationMinutes;
+  int postOrgasmDurationMinutes;
```

Comment:
For consistency, can you snake_case these? I think some tooling I have will get angry at camelCase.